packages/by-name/kata/contrast-node-installer-image/package.nix

```diff
@@ -40,62 +37,61 @@ let
           files = [
             {
               url = "file:///opt/edgeless/share/kata-containers.img";
-              path = "/opt/edgeless/${runtime-handler}/share/kata-containers.img";
+              path = "/share/kata-containers.img";
```

Comment:
Instead of just using the suffix here, should we use a placeholder to be more explicit instead?

Reply:
I don't really see the advantage. We control both the consumer and producer of this value and can align them. If it was a public API, I'd see the point, but I think this will just be unnecessary complexity here.

Reply:
The current design makes it impossible to place a file outside of /opt/edgelessys/....

Reply:
Since we don't need that right now?

I will just implement it, but I don't see the point in generalizing things too early that don't need to be general.

Reply:
How about making the paths relative? That would allow adding absolute paths, should we need them.

Reply:
> How about making the paths relative? That would allow adding absolute paths, should we need them.

I favor the explicit replacing in that case. I think having conditional behaviour depending on whether the path's relative or not might be confusing